Homework7/Car.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace Homework7
+{
+    public abstract class Car
+    {
+        public string? brand;
+        public string? model;
+        public int year;
+        public Car(string brand, string model, int year)
+        {
+            this.brand = brand;
+            this.model = model;
+            this.year = year;
+        }
+        public abstract int Speed { get; set; }
+        public abstract void EngineStart();
+        public void DisplayCarDetails()
+        {
+            Console.WriteLine($"{brand} {model} {year}");
+        }
+        public void Move()
+        {
+            Console.WriteLine("Wrooom!");
+        }
+    }
+    public class ElectroCar : Car
+    {        
+        public int batteryCapacity;
+        private int speed;
+        public ElectroCar (string brand, string model, int year, int batteryCapacity) : base (brand, model, year)
+        {
+            this.batteryCapacity = batteryCapacity;
+        }
+        public override int Speed { get { return 0; } set { speed = value; } }
```

Comment:
It should return private field speed. Or you can leave only autoproperty Speed {get; set;}

Reply:
updated - Speed { get { return speed; } set { speed = value; } }